gruntfile.coffee

```diff
@@ -173,7 +177,7 @@ module.exports = ->
   @registerTask 'clean', ['remove']
   @registerTask 'lint', ['coffeelint']
   @registerTask 'build', ['clean', 'coffeelint', 'concat:dist', 'string-replace:strip_for_dist', 'coffee:dist','string-replace:strip_asserts', 'uglify:dist']
-  @registerTask 'test', ['coffeelint', 'mochaTest:all']
+  @registerTask 'test', ['coffeelint', 'mochaTest:all', 'mochaTest:examples']
```

Comment:
I don't think this should be part of the test suite; it doesn't test anything new (or it shouldn't, anyways) and it therefor adds unnecessary time to running tests.